webview/src/shared/components/slider/MinMaxSlider.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FormEvent } from 'react'
```

Comment:
There appears to be little space between the plots and the slider in the custom and trends sections: 

<img width="1466" alt="Screenshot 2023-03-08 at 8 19 11 AM" src="https://user-images.githubusercontent.com/43496356/223737235-ab47e0e1-ab2e-4edd-8e66-c6dbef5f935c.png">